functions/src/index.ts

```diff
@@ -54,3 +54,5 @@ exports.sendMail = functions.https.onRequest((req: any, res: any) => {
     });
   });
 });
+
+export {};
```

Comment:
Hopefully this isn't problematic in the future

Reply:
yeah this was giving web a compile error without it, not sure why this fixes things